rpcs3/Emu/system_utils.cpp

```diff
@@ -136,19 +136,21 @@ namespace rpcs3::utils
 	{
 		const std::string home_dir = get_hdd0_dir() + "/home";
 
+		std::string rap_path;
+
 		for (auto&& entry : fs::dir(home_dir))
 		{
 			if (entry.is_directory && check_user(entry.name))
 			{
-				std::string rap_path = fmt::format("%s/%s/exdata/%s.rap", home_dir, entry.name, rap);
+				rap_path = fmt::format("%s/%s/exdata/%s.rap", home_dir, entry.name, rap);
 				if (fs::is_file(rap_path))
 				{
 					return rap_path;
 				}
 			}
 		}
 
-		return {};
+		return rap_path;
```

Comment:
at least comment why it is returned here

Reply:
Fixed.